src/test/java/edu/indiana/dlib/amppd/controller/WorkflowControllerTests.java

```diff
@@ -22,14 +21,15 @@
     private MockMvc mvc;
 
     // TODO remove @Ignore once Galaxy is set up for CI env
-    @Ignore
+//    @Ignore
     @Test
     public void shouldReturnWorkflows() throws Exception {
     	mvc.perform(get("/workflows")).andExpect(status().isOk()).andExpect(
     			// TODO need to import org.hamcrest.Matchers.hasSize with added dependency hamcrest-all, 
     			// TODO also need to add boolean expression for the case when no workflow exists
-//				jsonPath("$", hasSize(1))).andExpect(	
-						jsonPath("$[0].model_class").value("StoredWorkflow"));
+    			//jsonPath("$", hasSize(1))).andExpect(	
```

Comment:
I am getting, we will redo this, once Galaxy for testing is in place? also, we might have to create some workflows using api before testing.

Reply:
is there a way to bootstrap galaxy with some sample data? one workflow could be part of that sample data. a lot of unit tests touch galaxy will probably need such data. it would be best if there're some factory method to do this.